kong/api/routes/upstreams.lua

```diff
@@ -107,4 +109,54 @@ return {
       crud.post(self.params, dao_factory.targets)
     end,
   },
+
+  ["/upstreams/:name_or_id/targets/active/"] = {
```

Comment:
I won't get to it until monday, so if #2252 is ok and you got time, then please merge it. Rebase this and merge it also (just approved this one).